package.json

```diff
@@ -21,7 +21,6 @@
     "eslint-plugin-fxa": "git://github.com/mozilla/eslint-plugin-fxa.git#41504c9dd30e8b52900c15b524946aa0428aef95",
     "eslint-plugin-sorting": "git://github.com/shane-tomlinson/eslint-plugin-sorting.git#bcacb99d",
     "mocha": "10.4.0",
-    "request": "2.88.2",
```

Comment:
I think we need to update the package-lock.json as well.

But also, I did find one reference to the long dead "request" lib, so this might be a semi-breaking change:

```sh
git grep -Pin "[\"']request[\"']" | cat

loadtest/mailbox.js:4:const request = require('request');
package-lock.json:1905:    "request": {
```
 https://github.com/mozilla/restmail.net/blob/d054f45c0089044b8233b435c0adb2c77dff6d30/loadtest/mailbox.js#L4

Where the 2 usages are a GET on L56 and a DELETE on L79:

https://github.com/mozilla/restmail.net/blob/d054f45c0089044b8233b435c0adb2c77dff6d30/loadtest/mailbox.js#L56
https://github.com/mozilla/restmail.net/blob/d054f45c0089044b8233b435c0adb2c77dff6d30/loadtest/mailbox.js#L79

Not sure what to do with that. Either convert that to native `fetch()` which might mean bumping our min Node version (although looks like CircleCI is already pinned to 18.11) or possibly using some polyfill like node-fetch or a lib like axios/got.
Or if we want to get yolo with it, just delete the load tests if that isn't something we actively run.

Reply:
moved to https://github.com/mozilla/restmail.net/pull/136